### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alantsui5**
Add a line to notice user to install `nodemon` before booting up this template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ This is a simple template to get you started on writing Node.js applications usi
 ## Get started
 
 ```bash
+# if nodemon is not installed
+npm i -g nodemon
```
Comment (colloquet) [inline README.md:8]: Please add a blank line below just to keep it consistent.

**Comment by colloquet** [comment]
Hey Alan, thanks for the PR! I would suggest adding a blank line below line 8 just to keep it consistent 😄

**Comment by colloquet** [comment]
Or do you think it's better to add `nodemon` as `devDependency` in `package.json`?

**Comment by alantsui5** [comment]
I think global or dev dependency are both okay, but I choose global because I think most of the JS developers need `nodemon`. Sorry for late reply.

**Pull request closed without merging**